### PR TITLE
Add Tenant Operator Installation Process

### PR DESCRIPTION
**Pull request by abdulhaseeb2**

### CODE REVIEW

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-239-a69e18d6`

File: content/sre/tenant-operator/customresources.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 apiVersion: tenantoperator.stakater.com/v1alpha1
 kind: Tenant
 metadata:
-  name: development
+  name: blue-sky
```
Comment (rasheedamir) [inline content/sre/tenant-operator/customresources.md]: why not alpha (lol) anyways bluesky is without -

Comment (abdulhaseeb2) [reply content/sre/tenant-operator/customresources.md]: To honor TO's creator 😄

File: content/sre/tenant-operator/customresources.md [hunk 2]
```diff
@@ -1,18 +1,39 @@
...
+
+```yaml
+apiVersion: tenantoperator.stakater.com/v1alpha1
+kind: Quota
+metadata:
+  name: medium
+  annotations:
+    quota.tenantoperator.stakater.com/is-default: "false"
+spec:
+  hard:
```
Comment (rasheedamir) [inline content/sre/tenant-operator/customresources.md:19]: @abdulhaseeb2 add cpu and ram here which are the key metrics

File: content/sre/tenant-operator/installation.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+```bash
+oc apply -f -n stakater-tenant-operator docker-secret.yaml
+```
+
+Create a `docker-secret` in *stakater-tenant-operator* namespace to pull Tenant-Operator image from dockerhub.
+
+*The secret will be provided by **Stakater***
+
+## 3. Create Tenant-Operator Helm Release
```
Comment (rasheedamir) [inline content/sre/tenant-operator/installation.md]: @abdulhaseeb2 can we suggest another option to install via helm charts?

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-239-34b7e8d7`

File: content/sre/tenant-operator/installation.md (hunk 3 above)
Comment (abdulhaseeb2) [reply content/sre/tenant-operator/installation.md]: Sure let me test that out and I'll add it.

Comment (rasheedamir) [reply content/sre/tenant-operator/installation.md]: perfect! as most customers will like to deploy it via ArgoCD which has native helm charts support

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-239-cd02024a`

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-239-dfe151b1`

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-239-b762265a`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.177`

**Pull request merged**